stdlib/3.3/lzma.pyi

```diff
@@ -0,0 +1,105 @@
+import io
+import sys
+from typing import Any, Dict, BinaryIO, IO, List, Optional, Union
+
+if sys.version_info >= (3, 6):
+    from os import PathLike
+    _PathOrFile = Union[str, bytes, IO[Any], PathLike[Any]]
+elif sys.version_info >= (3, 3):
+    _PathOrFile = Union[str, bytes, IO[Any]]
+else:
+    _PathOrFile = str
+
+_FilterChain = List[Dict[str, Any]]
```

Comment:
Does this really have to be a List of Dicts? Is an Iterable of Mappings also acceptable?

Reply:
Good point; from the source it looks like anything dict-like should be fine; a824afe.

While we're thinking about this: the dictionary must always have at least an `'id'` key defined -- is there a way to add that to the type signature?

Reply:
Possibly with TypedDict, but it's fine to not worry about it for now.